src/EditorFeatures/Core/Implementation/IntelliSense/Completion/AsyncCompletionService.cs

```diff
@@ -72,6 +79,14 @@ public bool TryGetController(ITextView textView, ITextBuffer subjectBuffer, out
                 return false;
             }
 
+            var vsTextView = _editorAdaptersFactoryService.GetViewAdapter(textView);
+            var wpfTextView = _editorAdaptersFactoryService.GetWpfTextView(vsTextView);
+            if (!_featureServiceFactory.GetOrCreate(wpfTextView).IsEnabled(PredefinedEditorFeatureNames.Completion))
```

Comment:
Does this need to be moved inside of UseLegacyCompletion if there are other callers?